src/dropdown.js

```diff
@@ -182,9 +182,17 @@ export default class Dropdown extends EventEmitter {
 
   /** @private */
   setOffset(cursorOffset: CursorOffset) {
+    const elementWidth = this.el.offsetWidth;
     if (cursorOffset.left) {
+      const browserWidth = document.documentElement.clientWidth;
```

Comment:
`document.documentElement` is nullable. I think we should merge L187 and L199, for example:

```js
setOffset(cursorOffset: CursorOffset) {
  const doc = document.documentElement;
  if (!doc) { return; } // non browser
  const elementWidth = this.el.offsetWidth;
  if (cursorOffset.left) {
    const browserWidth = doc.clientWidth;
  // ...
  if (this.isPlacementTop()) {
    this.el.style.bottom = `${doc.clientHeight - cursorOffset.top + cursorOffset.lineHeight}px`
  // ...
}
```

Reply:
Done